sources/HUBUtilities.h

```diff
@@ -55,7 +55,7 @@ static inline BOOL HUBPropertyIsEqual(NSObject * _Nullable objectA, NSObject * _
  *
  *  This function asserts that a view has been loaded after -loadView was sent to the component.
  */
-static inline UIView *HUBComponentLoadViewIfNeeded(id<HUBComponent> component) {
+static inline UIView *HUBComponentLoadViewIfNeeded(id<HUBComponent> _Nullable component) {
```

Comment:
Right, we don't need this. I was sending bad object to this function and it led to following error
> Implicit conversion from nullable pointer to non-nullable pointer 
But now worries, I know how to fixed this 😉